emulator/src/main/java/dk/dbc/search/solrdocstore/emulator/Config.java

```diff
@@ -0,0 +1,82 @@
+package dk.dbc.search.solrdocstore.emulator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.ejb.EJBException;
+import javax.ejb.Startup;
+import javax.ejb.Stateless;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.ws.rs.core.UriBuilder;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+@Stateless
+@Startup
+public class Config {
+
+    private static final Logger log = LoggerFactory.getLogger(Config.class);
+
+    private String solrDocStoreUrl;
+    private UriBuilder solrDocStoreUriBuilder;
+
+    @PostConstruct
+    public void loadProperties() {
+        Properties props = findProperties("solr-doc-store-emulator-service");
+        solrDocStoreUrl = getValue(props, "solrDocStoreUrl", "SOLR_DOC_STORE_URL", null, "No solrDocStore url specified");
+        try {
+            solrDocStoreUriBuilder = UriBuilder.fromPath(solrDocStoreUrl);
+        } catch (IllegalArgumentException e) {
+            log.error("Error when resolving URL for SolrDocStore ({})", solrDocStoreUrl);
+            log.error("Error was: {}", e);
```

Comment:
Jeg tror den her linie matcher String, Exception og du derfor mangler et argument / har en {} for meget i din format string.